### ISSUE TITLE
Segmentation fault (core dumped)

### ISSUE THREAD

**Issue by gsgrippal** [Aorimn/dislocker#211]
Trying to open an encrypted partition
```
dislocker by Romain Coltel, v0.7.1 (compiled for Linux/x86_64)
```
```
dislocker-fuse -v -V /dev/disk/by-id/diskid-part3 -p[...] -- /mnt/windows_c/
Segmentation fault (core dumped)
```

**Comment by gsgrippal** [comment]
Actually happens with all encrypted partitions I've tested.

**Comment by gsgrippal** [comment]
Fixed upgrading ubuntu from 19.10 to 20.04 development branch. No idea where the bug was but fixed.
